app/jobs/articles/async_bust_job.rb

```diff
@@ -0,0 +1,11 @@
+module Articles
+  class AsyncBustJob < ApplicationJob
```

Comment:
I'd call this `BustCacheJob` to mimic the name of `Comments::BustCacheJob`. Async is also redudant, being this a job...